kernel/dev/rtc/rtc.cpp

```diff
@@ -69,15 +76,17 @@ uint8_t rtc_get_register(uint8_t reg) {
     Note: This uses the "read registers until you get the same values twice in a row" technique
     to avoid getting dodgy/inconsistent values due to RTC updates
 */
-void read_rtc() {
+void read_rtc()
+{
     // Previous values from RTC
     // Used as a cache to check if we should update
     uint32_t last_second = 0, last_minute = 0,
-            last_hour = 0, last_day = 0,
-            last_month = 0,  last_year = 0,
-            last_century = 0, registerB = 0;
+             last_hour = 0, last_day = 0,
+             last_month = 0, last_year = 0,
+             last_century = 0, registerB = 0;
     // Make sure an update isn't in progress
-    while (rtc_get_update_in_progress());
+    while (rtc_get_update_in_progress())
+        ;
```

Comment:
I know this is likely the formatter's fault, but could you revert this? (Put the semicolon on the same line as the while)